commands/features.go

```diff
@@ -0,0 +1,54 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ConradIrwin/font/sfnt"
+)
+
+// Features prints the gpos/gsub tables (contains font features).
+func Features() {
+	if len(os.Args) < 2 {
+		panic(fmt.Errorf("Specify a font file"))
+	}
+
+	file, err := os.Open(os.Args[1])
+	if err != nil {
+		panic(fmt.Errorf("Failed to open font: %s", err))
+	}
+	defer file.Close()
+
+	font, err := sfnt.Parse(file)
+	if err != nil {
+		panic(fmt.Errorf("Failed to parse font: %s", err))
+	}
+
+	layoutTable(font, sfnt.TagGsub, "Glyph Substitution Table (GSUB)")
+	layoutTable(font, sfnt.TagGpos, "Glyph Positioning Table (GPOS)")
+}
+
+func layoutTable(font *sfnt.Font, tag sfnt.Tag, name string) {
+	if font.HasTable(tag) {
+		fmt.Printf("%s:\n", name)
+
+		t := font.Table(tag).(*sfnt.TableLayout)
```

Comment:
@bramp This will panic if parsing fails. I'm not sure what to do about that except maybe change the library so that it parses tables on-demand instead of up-front so we can return a sensible error. Any better ideas?

Reply:
Yes, panic'ng sucks. After this I'd be happy to change the whole library to pass back the errors.

Reply:
That'd be incredible 😻